### ISSUE TITLE
fine-tuning on 26k

### ISSUE THREAD

**Issue by aqzhang218** [Lavreniuk/SPIdepth#13]
I would like to ask that SPIdepth conducts training on 70k and fine-tuning on 26k. The fine-tuning file I checked seems to use supervised training for fine-tuning. Does this violate the purpose of self-supervision?

**Comment by aqzhang218** [comment]
Thank you very much for coming back, but in the first step: self-supervised training and evaluation without indicators, you used a 70k data set, which is crossed with the data in the test set. Will this affect the evaluation results?

**Comment by Lavreniuk** [comment]
I just follow previous papers protocol on self-supervised training and evaluation for direct comparison of improvement.
But from my point of view as far as you did not use metric depth (labels) you could use them, because in any production system you have access to images, but not for depth maps, so you could fine tune it and get same quality.
